coordinator/src/routes/tracker.ts

```diff
@@ -0,0 +1,58 @@
+import Hapi from "@hapi/hapi"
+import {fhir} from "@models"
+import {BASE_PATH, ContentTypes} from "./util"
+
+interface lineItem {
+  id: string,
+  status: string
+}
+
+interface trackerResponse extends Partial<fhir.Task> {
+  lineItem: Array<lineItem>
+}
+
+const sandboxSuccessResponse = (prescriptionId: string): trackerResponse => ({
+  resourceType: "Task",
+  groupIdentifier: fhir.createIdentifier("system", prescriptionId),
```

Comment:
System should be "https://fhir.nhs.uk/Id/prescription-order-number" for short form prescription ID